app/models/product.rb

```diff
@@ -25,4 +25,8 @@ class Product < ApplicationRecord
   def display_image
     image.variant(resize_to_limit: [100, 100])
   end
+
+  def product_image
```

Comment:
Sử dụng config variant của active_storage, ko nên tạo mới method như này.

Tham khảo: https://edgeguides.rubyonrails.org/active_storage_overview.html#:~:text=You%20can%20configure%20specific%20variants%20per%20attachment%20by%20calling%20the%20variant%20method%20on%20yielded%20attachable%20object%3A

và xem lại ở bài giảng về active_storage (chap 13)